src/components/ConcentratedLiquidity/PositionCard/types.tsx

```diff
@@ -0,0 +1,6 @@
+import { Currency } from '@pangolindex/sdk';
+
+export type PositionCardProps = {
+  currency0: Currency;
+  currency1: Currency;
+};
```

Comment:
just some thought. I see that in components repo, in some components we keep types in separate file and in some component we keep types with component file. what do you think is better ? 
In my opinion we can keep types in separate file if that type is going to be imported somewhere. if that type is only for that specific component then we can keep it in same component file. what do you prefer ? 
whichever style we prefer, lets keep it consistent across all components.

Reply:
I prefer to keep `types.tsx` always seperate for each component. It helps to write proper story file. And keeps the code more clear/understandable. But for common using types (except directly component type), we can use common type file. (I've done similar thing for `Bridge`) `pbridge/types.tsx`

Reply:
make sense. lets follow same structure everywhere then from going forward.